repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilter.java

```diff
@@ -78,6 +78,8 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicReference;
 
+import static org.openrepose.commons.utils.http.CommonHttpHeader.REQUEST_ID;
```

Comment:
Should this be made an `_` and just bring them all in?
Then a simple find and replace could clean up a lot of code.